app/src/main/java/com/github/h3lp3rs/h3lp/database/Databases.kt

```diff
@@ -4,15 +4,25 @@ package com.github.h3lp3rs.h3lp.database
  * Enumeration of all useful databases in H3LP
  */
 enum class Databases {
-    PREFERENCES, EMERGENCIES, NEW_EMERGENCIES;
-    var db: Database = FireDatabase(name) // Var to enable test-time mocking
+    PREFERENCES, EMERGENCIES, NEW_EMERGENCIES, MESSAGES, CONVERSATION_IDS;
+    var db: Database? = null // Var to enable test-time mocking

```

Comment:
We could make the db attribute private now since we have a setter and a 'getter'

Reply:
Nice catch!